nmrfx-core/src/main/java/org/nmrfx/utilities/DatasetSummary.java

```diff
@@ -11,16 +11,15 @@
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
-import java.util.ArrayList;
 import java.util.List;
 
 /**
  * @author brucejohnson
  */
-public class RemoteDataset {
+public class DatasetSummary {
```

Comment:
That's a better name, it fixes the "Remote" used for local files.

However, we do have some files that are only FID, and some other parts of the code have this split in VendorFile/FID/Dataset with dataset being used only for processed files, so I'm not sure. I don't know if we want to keep this FID/Dataset terminology to indicate whether it's a raw or processed file.

Reply:
Ah true, and the method toDatasetSummary() is actually in NMRData not Dataset....maybe ExperimentSummary? but thats not a term that is used really in nmrfx

Reply:
or even just DataFileSummary, but then we can also ask ourselves why opening a DatasetBrowser if we don't open datasets. Maybe don't waste your time on this naming and keep DatasetSummary if you didn't already changed it then.

Reply:
Okay will leave as is for now, we can rename later